### PR TITLE
Using alias for interface members and && to split long texts

### PR DESCRIPTION
**Pull request by shafiq2601**

### CODE REVIEW

**Comment by shafiq2601** [comment]
Please review the sections I have added. I have observed the anti-pattern being practiced by a lot of developers. Let me know for any required modification.

File: clean-abap/CleanABAP.md [hunk 1]
```diff
@@ -2181,6 +2200,29 @@ which will never have an alternative implementation and will never be mocked in
...
 describes why this also applies to classes that overwrite inherited methods.
 
+#### Use alias for using interface members
+
+> [Clean ABAP](#clean-abap) > [Content](#content) > [Methods](#methods) > [Methods: Object orientation](#methods-object-orientation) > [This section](#use-alias)
+
+While calling interface methods or using interface attributes, using alias is much cleaner.
+
+```ABAP
+class clean_class definition public inheriting from if_clean_interface.
```
Comment (larshp) [inline clean-abap/CleanABAP.md:2210]: syntax error? `inheriting from if_clean_interface`

**Comment by larshp** [comment]
I see use of ALIASES as an anti-pattern, aliases become necessary due to bad OO design IMHO

**Comment by shafiq2601** [comment]
True. But when we inherit from legacy interfaces, the interface name is long sometimes and in that scenario, I believe aliases does help tidying up the code for me.

File: clean-abap/CleanABAP.md (hunk 1 above)
Comment (shafiq2601) [reply clean-abap/CleanABAP.md:2210]: I did not check the code in editor and i did not close the class definition. Is that why?

**Comment by larshp** [comment]
yea, then suggest adding a note that this is useful for legacy interfaces, and when designing new OO use of aliases should be minimized?

**Comment by shafiq2601** [comment]
@larshp: Did you also check "use && to split long texts" part?

File: clean-abap/CleanABAP.md (hunk 1 above)
Comment (mAltr) [reply clean-abap/CleanABAP.md:2210]: A class can not inherit from an interface in abap ;)

Comment (shafiq2601) [reply clean-abap/CleanABAP.md:2210]: My bad. I will correct the code. Thanks for pointing out.

**Pull request closed without merging**